packages/chat/tests/messageUtils.test.ts

```diff
@@ -0,0 +1,110 @@
+			'is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a',
+			'test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test'
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 2 messages', () => {
+		expect(autoSplitterWithSpaces(testStringNoSpaces.repeat(40), 250)).toStrictEqual([
+			'ThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisat',
+			'estThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatest'
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 2 messages', () => {
+		expect(autoSplitterWithSpaces(testStringNoSpaces.repeat(60), 500)).toStrictEqual([
+			'ThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisi',
+			'satestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatest'
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 3 messages', () => {
```

Comment:
We have a test for 3 messages and a test for no spaces, there's probably no need to test the combination.